cmd/podman/play_kube.go

```diff
@@ -206,6 +213,12 @@ func playKubeYAMLCmd(c *cliconfig.KubePlayValues) error {
 			// the other containers are in created status
 			return err
 		}
+	}
+
+	// We've now successfully converted this YAML into a pod
+	// print our pod and containers, signifying we succeeded
+	fmt.Println(pod.ID())
```

Comment:
Should there be header here?  To allow users to differentiate between pod and containers?

Reply:
How does that look?